edgelet/hosting/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "hosting"
+version = "0.1.0"
+authors = ["Azure IoT Edge Devs"]
+publish = false
+description = """
+Includes the code-generated implementation of the hosting
+api.
+"""
+edition = "2018"
+
+[dependencies]
+base64 = "0.9"
+failure = "0.1"
+futures = "0.1"
+hyper = "0.12"
+serde = "1.0"
+serde_derive = "1.0"
+serde_json = "1.0"
+serde_yaml = "0.7"
+typed-headers = "0.1"
+url = "1.5"
+
+[dev-dependencies]
+tokio-core = "*"
```

Comment:
Don't use `tokio-core`. It's the old tokio runtime. If you need a single-threaded runtime for tests, etc, use `tokio::runtime::current_thread::Runtime`. You can find examples in the rest of the codebase.

AFAICT you don't actually use `tokio_core` anywhere anyway.

Reply:
This crate was entirely created by swagger-codegen, not sure why it added that. Will check and remove it.